moto/rds/models.py

```diff
@@ -2225,6 +2245,9 @@ def __init__(self, region_name: str, account_id: str):
             GlobalCluster: self.global_clusters,
             OptionGroup: self.option_groups,
         }
+        self.bluegreen_deployments: MutableMapping[str, BlueGreenDeployment] = (
```

Comment:
minor nit: can we call this `blue_green_deployments` to keep in line with AWS naming?